src/clock.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+class Clock extends React.Component {
+  constructor(props) {
+    super(props);
+    this.timeZone = props.timeZone;
```

Comment:
this here is unnecessary. You should just be able to use `this.props` to access the props from anywhere in the component without defining those in the constructor